src/core/logger/DiagnosticBuilder.ml

```diff
@@ -26,9 +26,9 @@ struct
   let append_marks d marks =
     Diagnostic.{ d with additional_marks = d.additional_marks @ marks }
 
-  let tracef ?loc fmt =
+  let tracef ?loc tag fmt =
```

Comment:
Should we call everything `phase` or `tag`?

Reply:
it should be phase, I just forgot to change that one.